example/index.ts

```diff
@@ -1,20 +1,19 @@
-import elmStaticHtml from "../index";
 import * as fs from "fs";
+import * as elmStaticHtml from "../index";
 
 const model = { name: "Noah", age : 24 };
 const secondModel = { name: "not noah", age: 74};
 const firstRunOptions = { model : model, decoder: "MyModule.decodeModel", alreadyRun: false };
 const secondRunOptions = { model : secondModel, decoder: "MyModule.decodeModel", alreadyRun: true };
 
-
 function runTwice() {
-    elmStaticHtml(process.cwd(), "MyModule.view", firstRunOptions)
+    elmStaticHtml.default(process.cwd(), "MyModule.view", firstRunOptions)
     .then((generatedHtml) => {
         fs.writeFileSync("output.html", generatedHtml);
-        elmStaticHtml(process.cwd(), "MyModule.view", secondRunOptions)
+        elmStaticHtml.default(process.cwd(), "MyModule.view", secondRunOptions)
         .then((generatedHtml) => {
             fs.writeFileSync("output2.html", generatedHtml);
-            elmStaticHtml(process.cwd(), "MyModule.view", secondRunOptions)
+            elmStaticHtml.default(process.cwd(), "MyModule.view", secondRunOptions)
```

Comment:
This usage seems wrong,based on the changes you made. Doesn't it return an object now, like the multiple one? I'd rather it didn't return an object if it's a single view

Reply:
struggling a bit with typescripts module system. 

I think `elmStaticHtml` can't be the default and also be a module. Would it be OK to drop the default and just expose `elmStaticHtml` and `multiple`, and then import as 

```typescript
import { elmStaticHtml, multiple } from "elm-static-html-lib";
```